libs/langchain/tests/integration_tests/embeddings/test_vertexai.py

```diff
@@ -24,6 +24,70 @@ def test_embedding_query() -> None:
     assert len(output) == 768
 
 
+def test_large_batches() -> None:
```

Comment:
should it be a unit test?
there's an example how to add a unit test that requires a custom dependency: https://github.com/langchain-ai/langchain/pull/12200/files

integrations tests are only compiled, not run on every push, so I'd rather move unit tests to the corresponding folder
@baskaryan please, correct me if I'm wrong